src/object/authenticate.h

```diff
@@ -52,6 +53,41 @@ extern const char *AU_GRANT_CLASS_NAME;
 extern const char *AU_PUBLIC_USER_NAME;
 extern const char *AU_DBA_USER_NAME;
 
+/* the order to connect to db-hosts in databases.txt */
```

Comment:
Is there any reason to move here instead of db_admin.c? As far as I could find, they are not used anywhere else.